s3/replication/manager/src/s3replicationmanager/config.py

```diff
@@ -29,10 +29,26 @@ class Config:
     def __init__(self, configfile):
         """Initialise."""
         if configfile is None:
-            self.configfile = os.path.join(os.path.dirname(__file__),
-                                           '..', 'config', 'config.yaml')
+            # Default config file directory : ~/.cortxs3/manager
+            config_home = os.path.join(
+                os.path.expanduser('~'), '.cortxs3', 'manager')
+            configfile = os.path.join(config_home, 'config.yaml')
+            if os.path.isfile(configfile):
+                self.configfile = configfile
+                print(
```

Comment:
you can just use one print at end after path is resolved.

Reply:
done